riff-raff/app/schedule/DeployJob.scala

```diff
@@ -26,18 +26,22 @@ class DeployJob extends Job with Logging {
       uuid <- deployments.deploy(params, ScheduleRequestSource)
     } yield uuid
     result match {
-      case Left(error) => log.warn(error.message)
+      case Left(error) => {
+        //Send ang msg
```

Comment:
Is that comment still relevant?

Reply:
I think this is intended to capture the case where a scheduled deploy doesn't start at all, which is tracked by a separate card on our board: https://trello.com/c/3jb3GJvl/113-scheduled-deploys-dont-notify-when-they-cant-start

I'll clarify the comment.